### PR TITLE
Some Misc. changes

### PR DESCRIPTION
**Pull request by Rajat163**

### CODE REVIEW

File: src/main/java/com/rajat/mywebapp/controllers/CustomerController.java [hunk 1]
```diff
@@ -32,7 +32,7 @@ public List<CustomerDTO> getAllCustomers() {
         return service.getAllCustomers();
     }
 
-    @RequestMapping(params = "custId",method = RequestMethod.GET)
+    @GetMapping("customers/id")
```
Comment (drishti07) [inline src/main/java/com/rajat/mywebapp/controllers/CustomerController.java]: There is a mismatch in the parameter mentioned in the get mapping and in the function argument.

File: src/main/java/com/rajat/mywebapp/service/impl/CustomerServiceImpl.java [hunk 2]
```diff
@@ -23,14 +23,14 @@ public class CustomerServiceImpl implements CustomerService {
     private AddressServiceImpl addressService;
 
     @Override
-    public void addNewCustomer(CustomerDTO dto) {
+    public long addNewCustomer(CustomerDTO dto) {
```
Comment (drishti07) [inline src/main/java/com/rajat/mywebapp/service/impl/CustomerServiceImpl.java:26]: suggestion: put the code in try and catch blocks and handle exceptions.

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -2,6 +2,6 @@ spring.datasource.url=jdbc:mysql://localhost:3306/customer_manager
 spring.datasource.username=root
 spring.datasource.password=S@xena1216
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-spring.jpa.hibernate.ddl-auto=update
+spring.jpa.hibernate.ddl-auto=create
```
Comment (drishti07) [inline src/main/resources/application.properties]: I hope this is due to the fact that if you do any changes in db then it is dropped and created again?

File: src/main/java/com/rajat/mywebapp/controllers/CustomerController.java (hunk 1 above)
Comment (drishti07) [reply src/main/java/com/rajat/mywebapp/controllers/CustomerController.java]: Also if id is variable like from front end it should be like customers/24 then it should be like following:
@GetMapping("/{id}")

File: src/main/resources/application.properties (hunk 3 above)
Comment (Rajat163) [reply src/main/resources/application.properties]: Reverting this one for higher env.

**Pull request merged**